src/bregman.jl

```diff
@@ -111,19 +111,20 @@ function bregman(funobj::Function, x::AbstractArray{T}, options::BregmanParams,
         d .= -TD*g
         # Step length
         t = (options.spg && i> 1) ? T(dot(x-xold, x-xold)/dot(x-xold, g-gold)) : T(options.alpha*f/norm(d)^2)
+        t = abs(t)
```

Comment:
don't see how t can be not real or neagtive

Reply:
same as above